doc/howto/using_the_api.rst

```diff
@@ -115,6 +115,53 @@ you are using and exactly the value it should have::
 At the moment there is no way of specifying wildcards in the filter.
 
 
+Using POST, PUT, PATCH and DELETE
+---------------------------------
+
+To use these request methods you need a write-enabled token. Go to `User and API
+Administration </useradmin/tokens/>`_ to set token attributes.
+
+CRUD-methods are enabled for a limited number of endpoints. These endpoints can
```

Comment:
If I go to the root of the API, it still says

> The NAV API is currently read only.

Also in this very same doc page, under the **Tokens** section, it also claims the API is *read-only*

Reply:
I've done a very basic test of adding rooms and netboxes (but not deleting or changing them), and it all looks to work just fine.

However, I discovered some unrelated problems in the process, and I'm focusing on those. Just some feedback to my review, and some doc updates, and I'm sure we're good to go.